site/content/languages/_index.md

```diff
@@ -18,14 +18,62 @@ Ex. If your source is in Go, you might compile it with TinyGo.
 Below are notes wazero contributed so far, in alphabetical order by language.
 
 * [TinyGo](tinygo) Ex. `tinygo build -o X.wasm -target=wasi X.go`
+* [Rust](rust) Ex. `rustc -o X.wasm --target wasm32-wasi X.rs`
 
-wazero is a runtime that embeds in Golang applications, not a web browser. As
+wazero is a runtime that embeds in Go applications, not a web browser. As
 such, these notes bias towards backend use of WebAssembly, not browser use.
 
 Disclaimer: These are not official documentation, nor represent the teams who
 maintain language compilers. If you see any errors, please help [maintain][1]
 these and [star our GitHub repository][2] if they are helpful. Together, we can
 make WebAssembly easier on the next person.
 
+## Concurrency
+
+WebAssembly's virtual machine has no concept of processor or core. Similarly,
```

Comment:
Maybe "WebAssembly does not yet support true parallelism; it lacks support for multiple threads, atomics, and memory barriers.  (It may someday; see https://github.com/WebAssembly/threads.)"